pipeline_utils/lib/yaml_parser.py

```diff
@@ -636,22 +659,22 @@ def __init__(self, data):
 
     def to_json(
                self,
-               institution, # alias
-               project # alias
+               submission_centers, # alias list
+               consortia # alias list
                ):
         """Function to build the corresponding object in JSON format.
         """
         frmt_json = {}
 
         # common metadata
-        frmt_json[self.FILE_FORMAT_SCHEMA] = self.name
-        frmt_json[self.ALIASES_SCHEMA] = [f'{project}:{self.FILEFORMAT_TYPE_SCHEMA}-{self.name}']
-        frmt_json[self.INSTITUTION_SCHEMA] = self._link_institution(institution)
-        frmt_json[self.PROJECT_SCHEMA] = self._link_project(project)
+        frmt_json[self.IDENTIFIER_SCHEMA] = self.name
+        frmt_json[self.ALIASES_SCHEMA] = [f'{self._string_consortia(consortia)}:{self.FILEFORMAT_TYPE_SCHEMA}-{self.name}']
+        frmt_json[self.SUBMISSION_CENTERS_SCHEMA] = submission_centers
+        frmt_json[self.CONSORTIA_SCHEMA] = consortia
         frmt_json[self.DESCRIPTION_SCHEMA] = self.description
         frmt_json[self.STANDARD_FILE_EXTENSION_SCHEMA] = self.extension
-        frmt_json[self.VALID_ITEM_TYPES_SCHEMA] = getattr(self, self.FILE_TYPES_SCHEMA, ['FileReference', 'FileProcessed'])
-        frmt_json[self.EXTRAFILE_FORMATS_SCHEMA] = getattr(self, self.SECONDARY_FORMATS_SCHEMA, [])
+        # frmt_json[self.VALID_ITEM_TYPES_SCHEMA] = getattr(self, self.FILE_TYPES_SCHEMA, ['FileReference', 'FileProcessed'])
```

Comment:
This should still be needed no?

Reply:
Doug wants to try and remove this, I did comment it out for now